### PR TITLE
Update SDK docs

### PR DESCRIPTION
**Pull request by vonageCI**

### CODE REVIEW

File: .repos/nexmo/conversation-docs/ios/2.1.0/index.html [hunk 1]
```diff
@@ -189,73 +189,17 @@
         <section>
           <section class="section">
             
-            <h1 id='welcome-to-nexmoclient-sdk-ios-nxm' class='heading'>Welcome to NexmoClient SDK iOS (NXM)!</h1>
+            <h1 id='nexmo-39-s-mobile-sdk-for-ios' class='heading'>Nexmo&rsquo;s Mobile SDK for iOS</h1>
```
Comment (tbedford) [inline .repos/nexmo/conversation-docs/ios/2.1.0/index.html]: I would prefer this as "Nexmo Client SDK for iOS" as that is the consistent naming convention we've been using.

File: .repos/nexmo/conversation-docs/ios/2.1.0/index.html [hunk 2]
```diff
@@ -189,73 +189,17 @@
...
+<h2 id='setup' class='heading'>Setup</h2>
 
-<p align="center">
-    <img src="https://github.com/vonage/nexmo-sdk-ios/blob/develop/Utils/SDKClassDiagram.png" title="Class diagram">
-</p>
+<p>To get started with the nexmo SDK for iOS, check out the <a href="https://developer.nexmo.com/client-sdk/setup/add-sdk-to-your-app/ios">nexmo Mobile Developer Guide for iOS</a></p>
 <h2 id='license' class='heading'>License</h2>
 
-<p>Copyright &copy; 2018 Vonage. All rights reserved. Licensed only under the Nexmo SDK License Agreement (the <q>License</q>) located at <a href="license-link">License file</a>.</p>
+<p>Copyright &copy; 2018 Vonage. All rights reserved. Licensed only under the Nexmo SDK License Agreement (the <q>License</q>) located at License file.</p>
```
Comment (tbedford) [inline .repos/nexmo/conversation-docs/ios/2.1.0/index.html]: This should be: 

Copyright © 2018 - 2020 Vonage. 

In other words we should update the copyright notice to include the current year.

**Comment by tbedford** [comment]
@dimitrisniras flagging in case not seen.

**Comment by dimitrisniras** [comment]
> @dimitrisniras flagging in case not seen.

@ChenLev @horothesun That's for you I believe!

**Pull request merged**